src/components/ec-input-field/ec-input-field.spec.js

```diff
@@ -45,6 +45,11 @@ describe('EcInputField', () => {
     expect(wrapper.element).toMatchSnapshot();
   });
 
+  it('renders properly with the given prop autocomplete', () => {
```

Comment:
We need to test a positive case too, when we set input autocomplete to true.

Reply:
done